### ISSUE TITLE
edit last message

### ISSUE THREAD

**Issue by ohhmm** [RetroShare/RetroShare#906]
all IM support edit last message feature, please, implement it

**Comment by felisucoibi** [comment]
u mean after sending it??? edit it?

**Comment by ohhmm** [comment]
Yes

**Comment by sehraf** [comment]
Yes by sending a new one that should replaces the old one. The client can decide whether he replaced the old with the new one or shows both. (eg conversation has these options)

**Comment by ohhmm** [comment]
usually in all IM up arrow starts editing of last sent by the user message

**Comment by alfinal** [comment]
FYI this feature is working in forums mode (I think you are talking about chat in direct contact mode).

**Comment by felisucoibi** [comment]
yes, he means in chats to be able to edit or more than edit, delete latest message, is something now common in telegram and whatsapp
